api/.dockerignore

```diff
@@ -7,4 +7,9 @@ grib
 .vscode
 **/__pycache__
 **/*.pyc
-python_cache
\ No newline at end of file
+python_cache
+junk/**/*
+junk
```

Comment:
Do we want to include `junk` in this list? I presume it's a Sy-specific local folder?

Reply:
It is Sy specific. I strongly advocate that we ignore my junk folder.